src/features/home/pages/GenericHome.native.test.tsx

```diff
@@ -152,12 +156,44 @@ describe('GenericHome page - Analytics', () => {
 
     await waitFor(() => {
       expect(analytics.logAllModulesSeen).toHaveBeenCalledWith(1)
-      expect(BatchUser.trackEvent).toHaveBeenCalledWith('has_seen_all_the_homepage')
+      expect(BatchUser.trackEvent).toHaveBeenCalledWith(
+        'has_seen_all_the_homepage',
+        undefined,
+        // we receive the empty BatchEventData mock
+        expect.anything()
+      )
+      expect(put).toHaveBeenCalledWith('home_id', 'fake-id')
+      expect(put).toHaveBeenCalledWith('home_type', 'mainHome')
```

Comment:
Si tu modifies le mock comme au dessus, tu pourras écrire un test plus compréhensible, plus orienté métier et moins technique
```suggestion
      expect(BatchUser.trackEvent).toHaveBeenCalledWith('has_seen_all_the_homepage', undefined, {
        home_id: 'fake_id',
        home_type: 'mainHome',
      })
```